openstudiocore/src/airflow/Airflow.i

```diff
@@ -26,6 +25,27 @@
 
 %ignore IndexModelImpl;
 %ignore IndexModel(Reader &input);
+%template(OptionalContamIndexModel) boost::optional<openstudio::contam::IndexModel>;
+
+// All the vectors
+%template(ContamSpeciesVector) std::vector<openstudio::contam::Species>;
+%template(ContamLevelVector) std::vector<openstudio::contam::Level>;
+%template(ContamDayScheduleVector) std::vector<openstudio::contam::DaySchedule>;
+%template(ContamWeekScheduleVector) std::vector<openstudio::contam::WeekSchedule>;
+%template(ContamWindPressureProfileVector) std::vector<openstudio::contam::WindPressureProfile>;
+%template(ContamPressureCoefficientPointVector) std::vector<openstudio::contam::PressureCoefficientPoint>;
+%template(ContamPlrTest1Vector) std::vector<openstudio::contam::PlrTest1>;
+%template(ContamPlrTest2Vector) std::vector<openstudio::contam::PlrTest2>;
+%template(ContamCvfDatVector) std::vector<openstudio::contam::CvfDat>;
+%template(ContamSimpleAhsVector) std::vector<openstudio::contam::Ahs>;
+%template(ContamZoneVector) std::vector<openstudio::contam::Zone>;
+%template(ContamAirflowPathVector) std::vector<openstudio::contam::AirflowPath>;
+%template(IntVectorVector) std::vector<std::vector<int> >;
```

Comment:
@jasondegraw maybe this could go here instead:

https://github.com/NREL/OpenStudio/blob/8296251db964cfdac272468381950d6327b42500/openstudiocore/src/utilities/core/CommonImport.i#L20